### ISSUE TITLE
test.sh problems

### ISSUE THREAD

**Issue by cvvergara** [MobilityDB/MobilityDB#130]
Copied from here: https://github.com/MobilityDB/MobilityDB/pull/129#issue-703569233 but only mentions what are the intended task, without mentioning what is the problem.

Three changes are needed in the test/scripts/test.sh file.
1- To make MobilityDB used by the test files, we need to set the shared_preload_libraries configuration parameter to Postgis-2.5.so.
2- To load the prerequisites tables, we need to get the load.sql.xz file from the source of the project. The change is in the following three files: test/general/CMakeLists.txt, test/point/CMakeLists.txt, test/npoint/CMakeLists.txt.
3- To start Postgres, the parameter run_ctl needs to be replaced with $PGCTL to get the complete command parameters and to print a message in the log file that is used to indicate whether PostgreSQL has started or not.

**Comment by cvvergara** [comment]
A comment on a PR about the issue
https://github.com/MobilityDB/MobilityDB/pull/129#issuecomment-893808208

**Comment by cvvergara** [comment]
```
SOFILE=$(echo "$BUILDDIR"/lib*.so)
```

The library name in windows its not ".so" in all systems/compilers, for example using Visual Studio on Windows, with pgrouting:
```
  -- Install configuration: "Release"
  -- Installing: C:/PROGRA~1/POSTGR~1/9.6/lib/pgrouting-3.2.lib
  -- Installing: C:/PROGRA~1/POSTGR~1/9.6/lib/libpgrouting-3.2.dll
  -- Installing: C:/PROGRA~1/POSTGR~1/9.6/share/extension/pgrouting--3.2.0--3.2.1.sql
  -- Installing: C:/PROGRA~1/POSTGR~1/9.6/share/extension/pgrouting--3.1.3--3.2.1.sql
  ...
```

**Comment by estebanzimanyi** [comment]
Solved with PR #138 and PR #131
